### PR TITLE
Add a Dockerfile

### PR DESCRIPTION
**Pull request by FireMasterK**
The next step would be to add a github actions workflow to build and push this image somewhere.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:3-slim
+
+WORKDIR /app/
+
+COPY requirements.txt .
+
+RUN --mount=type=cache,target=/root/.cache/pip \
+    pip install -r requirements.txt
```
Comment (unixfox) [inline Dockerfile:8]: Can you use `pip install --no-cache -r requirements.txt` instead?

**Review by unixfox — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (FireMasterK) [reply Dockerfile:8]: Don't we want to use the cache mount, though? There wouldn't be any point of the cache mount then. :thinking:

Comment (unixfox) [reply Dockerfile:8]: The issue is that the cache will be included inside the final docker image though.

Comment (FireMasterK) [reply Dockerfile:8]: This isn't true, the cache mount isn't included in the image.
image

**Comment by unixfox** [comment]
Could you work on the github actions workflow :)?

I'll configure the github secrets after.

**Pull request merged**